### ISSUE TITLE
Overlapping of code block and editor toolbar

### ISSUE THREAD

**Issue by Arash1381-y** [calumk/editorjs-codeflask#2]
I have tried to use this code block but when i want to use options something like this happens
![image](https://user-images.githubusercontent.com/79264867/189548338-ecca7577-7eff-4526-bb79-2af1551bd26a.png)

is there any way to fix it?

**Comment by yiwanbaimifan** [comment]
.codeflask.codeflask--has-line-numbers:before, .codeflask__lines{
    z-index: 0!important;
}

**Comment by asevarik** [comment]
it's half working 
![Screenshot 2023-03-10 at 10 41 47 PM](https://user-images.githubusercontent.com/70943186/224379688-80e2df20-f3a0-4eb1-97d8-d4a5456e407f.png)

**Comment by calumk** [comment]
Sorry i dont have time to look into this at this moment - if you find a fix please raise a PR or comment here, and i can try to impliment it

**Comment by calumk** [comment]
looks like adding 
```
 .ce-toolbar{
    z-index: 1000;
  }

````

resolves this, but its a change to the css of the editorjs not the plugin, so its not a great fix - ill look for a better one

**Comment by CKylinMC** [comment]
and also...
![image](https://github.com/calumk/editorjs-codeflask/assets/11648014/f0abd1f4-910f-4ccd-9563-b92e986c6a44)

the language select drop down maybe should have a larger space to show?

**Comment by calumk** [comment]
I agree - the issue is that editorjs doesnt expose any native dropdown tool, it only provides buttons - so this is a extra dependancy in the project. 
Ideally editorjs would provide a native dropdown, or i may depreciate it in favour of a non-styled dropdown, that is not as pretty, but more lightweight.

I dont have time to do this at the moment, someone is welcome to PR

**Comment by CKylinMC** [comment]
I tried to add two lines of styles to solve the overlapping issue:
```css
.editorjs-codeFlask_Wrapper{
    /* ... */
    z-index: 0;
    position: inherit;
}
```

![image](https://github.com/calumk/editorjs-codeflask/assets/11648014/339829b9-26d9-4784-a5e1-c9d604f91788)


And due to built-in filtering function works correctly, I flat all languages selects into the menu.

https://github.com/CKylinMC/editorjs-codeflask/commit/0fd8d1384606bbff8a18cce4ee8a695659efc595

![image](https://github.com/calumk/editorjs-codeflask/assets/11648014/7a36b086-bdb2-4771-9e7b-8eb388fc39ad)
![image](https://github.com/calumk/editorjs-codeflask/assets/11648014/c583d241-2fb9-4171-977e-ed28995405be)


(currently selected state not working properly)

**Comment by calumk** [comment]
> [!IMPORTANT]
> This repo will be archived soon. Please upgrade to [@calumk/editorjs-codecup](http://github.com/calumk/editorjs-codecup)

This new repo fixes this (any other issues), and removes dependancy on codeflask, instead using codecup
